learnable/src/components/CreateUserForm/CreateUserForm.jsx

```diff
@@ -80,7 +80,7 @@ const postData = async () => {
                             </div>
                              <div className="new-workshop-4a">
                                 <label htmlFor='Email'>Email:</label>
-                                <input onChange={handleChange} type="text" id='username' placeholder='Enter email'></input>
+                                <input onChange={handleChange} type="username" id='username' placeholder='Enter email'></input>
```

Comment:
We can't have an input type of username - here is a list of what we are able to use https://www.w3schools.com/html/html_form_input_types.asp